File: Blind150/1.Array&Hash/GO/contains_duplicate.go
// https://leetcode.com/problems/contains-duplicate/

package main


func containsDuplicate(nums []int) bool {
    seen := make(map[int]bool)

	for _, n := range nums {

		if seen[n] {
			return true
		}else{
			seen[n] = true
		}
	}
    		return false

}

File: Blind150/1.Array&Hash/Python/contains_duplicate.py
# https://leetcode.com/problems/contains-duplicate/

from typing import  List

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:

        seen = set()

        for n in nums:
            if n in seen:
                return True
            else:
                seen.add(n)
        return False
